plugins/telescope/default.nix

```diff
@@ -35,6 +35,18 @@ in
       description = "Configuration for the extensions. Don't use this directly";
       default = { };
     };
+
+    defaultsConfig = mkOption {
```

Comment:
Wouldn't `defaultConfig` be better than `defaultsConfig`?

Reply:
What about just `defaults`? It is closer to the native config.

Reply:
sure!